src/components/screens/LunchSection.tsx

```diff
@@ -181,10 +192,20 @@ export const LunchSection = ({ lunchOffers = lunchOffersDummy, navigation }: Pro
         </TouchableOpacity>
       )}
       <RegularText />
+      {!!additionalText && (
+        <>
+          <RegularText>{additionalText}</RegularText>
+          <RegularText />
+        </>
+      )}
+      {!!lunchOffers && (
+        <>
+          {lunchOffers.map((item, index) => (
+            <LunchOffer key={index} name={item.name} price={item.price} />
+          ))}
+        </>
+      )}
```

Comment:
this would work without the fragment, right?

```
{!!lunchOffers &&
  lunchOffers.map((item, index) => (
    <LunchOffer key={index} name={item.name} price={item.price} />
  ))}
```

Reply:
you are right. i removed the fragment in 3795523c9fe7f462df6678bc1c684c0c58969914